src/docs/release-notes/R25/FullNotes.mdx

```diff
@@ -0,0 +1,38 @@
+### Angular and React
+
+-   Migrated all Brightlayer UI components' documentation from Storybook to an in-house web app! We included more examples and more detailed usage explanations. Learn more about the documentation here, and [let us know](/community/contactus) what you think:
```

Comment:
"Migrated all Brightlayer UI components' documentation from Storybook to an in-house web app, which includes more context around how to use our components, example code snippets & live code-generation!" 

Just a thought, wanted to include shoutouts to 1. Context 2. Code Gen & 3. Updated Code Snippets since these are all new features that previously were not included in storybook.  Technically SB had code snippets, but they were littered with storybook decorations.  Now our sample code is just the component.

Reply:
I would need full English support on this... I added a blurb that says "for a better user experience"

Reply:
How about... (this is a little long?)

"Migrated our component library documentation site from Storybook to an in-house solution, which gave us full customization around how our docs behave.   We have added some powerful new features (to our component docs), such as an Examples page that provides accurate code snippets and context for each component we support.  We also introduced a Playground section, which generates a working, copy/pastable snippet based on user input."